Makefile

```diff
@@ -345,6 +355,13 @@ promote-agbot:
 
 promote-mac-pkg-and-docker: promote-mac-pkg promote-docker promote-agbot
 
+anax-k8s-image:
+	@echo "Producing ANAX K8S docker image $(ANAX_K8S_IMAGE)"
+	cp $(EXECUTABLE) $(ANAX_K8S_CONTAINER_DIR)/bin
+	cp $(CLI_EXECUTABLE) $(ANAX_K8S_CONTAINER_DIR)/bin
```

Comment:
Also, you need to add that /bin directory to .gitignore. See below, the binary is in the PR, which should not be the case.